velox/docs/functions/presto/math.rst

```diff
@@ -255,6 +255,12 @@ Probability Functions
     The successProbability must be real value in [0, 1], numberOfTrials and value must be
     positive integers with numberOfTrials greater or equal to value
 
+.. function:: inverse_beta_cdf(a, b, p) -> double
```

Comment:
In the Presto documentation these functions are separated into 2 lists "Probability functions : cdf" and "Probability functions : inverse_cdf". Lets follow the same in Velox docs also.

Reply:
Here is a link to the Presto doc https://prestodb.io/docs/current/functions/math.html#probability-functions-cdf

Reply:
Done :). Ready for re-review.